docs/contributing/contributing-releases/README.md

```diff
@@ -1,115 +1,156 @@
+   ```
+   git clone git@github.com:radius-project/radius.git
+   ```
+
 1. Create a new branch from `main`.
-1. In your local branch, update the `versions.yaml` file in the project-radius/radius repo
-The `versions.yaml` file is a declarative version tracking file that the Radius community maintains. Update it to reflect the new release candidate version that we would like to release. ([Example](https://github.com/project-radius/radius/pull/6077/files))
+   ```
+   git checkout main
+   git checkout -b <USERNAME>/<BRANCHNAME>
+   ```
+
+1. In your local branch, update the `versions.yaml` file to to reflect the new release candidate version that we would like to release. The `versions.yaml` file is a declarative version tracking file that the Radius community maintains ([Example](https://github.com/radius-project/radius/pull/6077/files)).
+
 1. Push these changes to a remote branch and create a pull request against `main`.
+   ```
+   git push origin <USERNAME>/<BRANCHNAME>
+   ```
+
 1. After maintainer approval, merge the pull request to `main`.
-1. Verify that [GitHub actions triggers a build](https://github.com/project-radius/radius/actions), and that the build completes. This will build and push Radius assets to [GHCR](https://github.com/orgs/radius-project/packages)
-1. In the project-radius/radius repo, run the [Release verification](https://github.com/radius-project/radius/actions/workflows/release-verification.yaml) workflow. Run the workflow from the release branch (format: release/x.y) and use the Radius RC release version number being released.
 
-### Test tutorials and samples
+1. You may need to wait around ~20 minutes for the release assets to be built and published.
+
+1. There should be a GitHub workflow run in progress [here](https://github.com/radius-project/radius/actions/workflows/build.yaml) that was triggered by the `vx.y.z-rc1` tag. Monitor this workflow to ensure that it completes successfully. If it does, then the release candidate has been created.
+
+1. Verify that a patch release was created on Github Releases for the current patch version ([Example](https://github.com/radius-project/radius/releases)).
+
+1. In the `radius-project/radius` repo, run the [Release verification](https://github.com/radius-project/radius/actions/workflows/release-verification.yaml) workflow. Run the workflow from the release branch (format: `release/x.y`) and use the Radius RC release version number being released.
 
-In the project-radius/samples repo, run the [Test Samples](https://github.com/project-radius/samples/actions/workflows/test.yaml) workflow. Run the workflow from the edge branch and using the Radius release or pre-release version number being released.
+1. In the `radius-project/samples` repo, run the [Test Samples](https://github.com/radius-project/samples/actions/workflows/test.yaml) workflow. Run the workflow from the `edge` branch and using the Radius release or pre-release version number being released.
```

Comment:
fix numbering

Reply:
I put all 1s here because markdown will do auto-numbering: https://github.com/radius-project/radius/blob/19715eea91bd1559078d1a8d496c40d977a556c9/docs/contributing/contributing-releases/README.md#creating-an-rc-release